.golangci.yaml

```diff
@@ -0,0 +1,63 @@
+# This is for linting. To run it, please use:
+# golangci-lint run ${MODULE}/... [--fix]
+
+linters:
+  # linters to run in addition to default ones
+  enable:
```

Comment:
I think it would be useful to have this configuration in a `.golangci.yaml` config file at the root of the repo. That would make it easier to replicate the CI results locally. (maybe instead of in the CI script)

Reply:
This is the case, no?

```
tricky ~/Dev/go-pam
❯ git log .golangci.yaml  
commit 40036cce8a6eeab097a6767146d321951c1319d1 (mine/add-ci-lint, add-ci-lint)
Author: Marco Trevisan (Treviño) <mail@3v1n0.net>
Date:   Tue Nov 7 11:51:27 2023 +0200

    ci: Use golang-ci linter
```

Reply:
Yes, my mistake. For some reason I though this was part of the CI config.